pkg/controller/migmigration/migrate.go

```diff
@@ -22,6 +22,9 @@ import (
 	vrunner "github.com/fusor/mig-controller/pkg/velerorunner"
 )
 
+var migrateAnnotationKey = "openshift.io/migrate-copy-phase"
```

Comment:
If you guys have any suggestions on a way to share this as a constant between the two controllers please let me know. I don't think putting it in the `migapi` package makes sense so wasn't sure where else to put it.

Reply:
How about a "common" package (or something similar) like we do with the velero plugin?

Reply:
We've deliberately stayed away from packages and .go files named _common_, _util_, _helper_ etc thinking it's better to use meaningful names.  Also, to avoid them being a catch for things that have appropriate homes.

If this annotation is common for both stage and migration.  how about having the `Task` inject it?

Reply:
So the existence of the annotation is common, but the value is different in stage and migration.